theme/templatetags/dashboard_tags.py

```diff
@@ -51,13 +51,11 @@ def resource_verbose_name_builder(rsc_type):
 class ResourceNameDBToUIMap:
     dict = {
         'CompositeResource': 'composite',
-        'Time Series': 'timeseries',
         'ToolResource': 'webapp',
         'CollectionResource': 'collection'
     }
     dict_verbose_name = {
-        'CompositeResource': 'Composite',
-        'Time Series': 'Time Series',
-        'ToolResource': 'Web App',
+        'CompositeResource': 'Resource',
```

Comment:
do we need to edit Types here:
https://github.com/hydroshare/hydroshare/blob/4853-resource-type-name-fix/theme/templates/pages/group-unauthenticated.html#L195

Reply:
We don't need to change it here.